util/gather_git_credits.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+import subprocess
+import re
+from collections import defaultdict
+
+codefiles = r"(\.[ch](pp)?|\.lua|\.md|\.cmake|\.java|\.gradle|Makefile|CMakeLists\.txt)$"
+
+# two major versions back, for "Active Contributors"
+revs = "5.2.0..HEAD"
+# all time, for "Previous Contributors"
+#revs = "HEAD"
```

Comment:
I suggest refactoring this into a function, and calling it twice

Reply:
It now does the whole job including deduplicating contributors so they only appear once in "active" or "previous"